src/components/ha-navigation-picker.ts

```diff
@@ -124,89 +91,65 @@ export class HaNavigationPicker extends LitElement {
 
     const panelViewConfig = new Map(viewConfigs);
 
-    this.navigationItems = [];
+    this._navigationItems = [];
 
     for (const panel of panels) {
-      this.navigationItems.push(createPanelNavigationItem(this.hass!, panel));
+      const path = `/${panel.url_path}`;
+      const primary = getPanelTitle(this.hass, panel) || path;
+      this._navigationItems.push({
+        id: path,
+        primary,
+        secondary: primary ? path : undefined,
```

Comment:
`primary` is already `getPanelTitle(this.hass, panel) || path` so this ternary is redundant